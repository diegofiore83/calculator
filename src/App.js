import React from 'react';
import { makeStyles } from '@material-ui/core/styles';
import Calculator from './components/Calculator';

const useStyles = makeStyles(theme => ({
  app: {
    textAlign: 'center',
  },
  appCore: {
    backgroundColor: theme.palette.primary.main,
    minHeight: '100vh',
    display: 'flex',
    flexDirection: 'column',
    alignItems: 'center',
    justifyContent: 'center',
    fontSize: 'calc(10px + 2vmin)'
  }
}));

const Component = () => {
  const classes = useStyles();

  return (
    <div className={classes.app}>
      <div className={classes.appCore}>
        <Calculator />
      </div>
    </div>
  );
};

Component.displayName = 'App';

Component.propTypes = {};

Component.defaultProps = {};

export default Component;
