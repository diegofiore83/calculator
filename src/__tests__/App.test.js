import React from 'react';
import {shallow} from 'enzyme';
import App from '../App';

describe('App', () => {
  const props = {};

  let wrapper;

  beforeEach(() => {
      wrapper = shallow(<App />);
  });

  describe('render', () => {
      it('should render correctly', () => {
          expect(wrapper).toMatchSnapshot();
      });
  });
});
