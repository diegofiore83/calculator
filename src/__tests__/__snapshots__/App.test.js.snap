// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App render should render correctly 1`] = `
<div
  className="makeStyles-app-1"
>
  <div
    className="makeStyles-appCore-2"
  >
    <Calculator />
  </div>
</div>
`;
